features/bootstrap/context_request.go

```diff
@@ -199,12 +200,12 @@ func (c *requestContext) requestJWTTokenIsNotSet() error {
 
 func (c *requestContext) requestJWTTokenIsValidAdminToken() error {
 	jwtConfig := jwt.NewGuard(c.adminCred)
-	accessToken, err := jwt.IssueAdminToken(jwtConfig.SigningMethod, make(map[string]interface{}), jwtConfig.Timeout)
+	accessToken, err := jwt.IssueAdminToken(jwtConfig.SigningMethod, jwtgo.MapClaims{}, jwtConfig.Timeout)
 	if nil != err {
 		return fmt.Errorf("Failed to issue JWT: %v", err)
```

Comment:
`errors.Wrap(err, "Failed to issue JWT")`

Reply:
this are tests... I'm not willing to change it to use a library for errors

Reply:
Ah indeed, I didn't notice it was tests